content-src/experiments/email-tabs.yaml

```diff
@@ -0,0 +1,75 @@
+id: 20
+title: 'Email Tabs'
+slug: email-tabs
+is_featured: false
+platforms: ['addon']
+min_release: 62
+thumbnail: /static/images/experiments/email-tabs/email-tabs.png
+description: >
+  Easily create emails from all of your tabs and make saving and sharing easier than ever.
+introduction: >
+  <p>Ever needed to save or share a whole bunch of tabs as you research, shop, or just browse the web? Email tabs lets you create beautiful emails from your open tabs to save them for later or share them. You can use Email Tabs to automatically send along links, screenshots, or even the text from articles.<p>
```

Comment:
Looks like you're inconsistently using *Email Tabs* (most of the time) and *Email tabs*

Reply:
good catch

Reply:
@flodolo and @pdehaan  I think I got all the inconsistencies, let me know if you see any others.